client/packages/common/src/ui/layout/tables/components/index.tsx

```diff
@@ -30,14 +27,26 @@ export const BasicCell = <T extends RecordWithId>({
         textOverflow: 'ellipsis',
       }}
     >
-      {column.formatter(column.accessor({ rowData }), {
-        t: localisedText,
-        d: localisedDate,
-      })}
+      {children}
     </div>
   </Box>
 );
 
+export const BasicCell = <T extends RecordWithId>({
```

Comment:
added this, so that I could make use of the basic cell layout without recreating it

Reply:
In the `StockDonorColumn` I'm using the same cell layout and rather than just duplicating that, I've used the `BasicCellLayout` which is exported here, so:

```
  return <BasicCellLayout>{data?.data ?? ''}</BasicCellLayout>;
```

I wanted the same border radius and padding and text overflow that the basic cell has, so that the new cell looks the same. I also want the plugin cell to be updated if the base properties ever get changed